spec/rubocop/cop/rspec/string_as_instance_double_constant_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::StringAsInstanceDoubleConstant,
+               :config do
+  context 'when using a class for instance_double' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        instance_double(Foo, failure_message, stubs)
```

Comment:
```suggestion
        instance_double(Shape, area: 12)
```
The second optional positional arg is a "name", but I don't see this frequently used in practice.
The last one, a hash, is what is stubbed.

Let's give here and in the cop doc examples that are more similar to real usage.